### PR TITLE
Feature/issue 14

### PR DESCRIPTION
**Pull request by crbrox**
Add  version checking

Also, decrease log level for rule management

@fgalan @jcalderin 

Closes issue #14

### CODE REVIEW

File: documentation/admin.md [hunk 1]
```diff
@@ -28,14 +28,18 @@ that should return a 200 OK  and a body similar to
...
+            <dd>0.1.1</dd>
+        </dl>
     </body>
 </html>
 ```
 
 After a change in the configuration file, a configuration reload can be asked for with
 ```
-curl -v localhost:8080/perseo-core/index.jsp
+curl -v localhost:8080/perseo-core/reload.jsp
```
Comment (fgalan) [inline documentation/admin.md]: This is a functional change? Or just documentation that went obsolete and you have take the oportunity to update it?

File: documentation/admin.md [hunk 2]
```diff
@@ -44,3 +48,11 @@ reload: true
 ```
 in its content
 
+What version is deployed can be checked with
+```
+curl -v localhost:8080/perseo-core/version.jsp
```
Comment (fgalan) [inline documentation/admin.md]: Would it be possible to have the URL without '.jsp' at the end? To align with how the operation works in Orion, Cygnus, etc.

File: pom.xml [hunk 3]
```diff
@@ -71,6 +71,15 @@
     </dependencies>
 
     <build>
+        <resources>
+            <resource>
+                <directory>src/main/resources</directory>
+                <includes>
+                    <include>version.properties</include>
```
Comment (fgalan) [inline pom.xml]: So I understand that you are using at the end a version.propertines.

That's fine but... just for curiosity, which technique is using @frbattid for implementing /version in Cygnus?

File: documentation/admin.md (hunk 1 above)
Comment (crbrox) [inline documentation/admin.md]: Exactly, that piece of documentation was wrong.

File: pom.xml [hunk 4]
```diff
@@ -117,49 +126,49 @@
                     </execution>
                 </executions>
             </plugin>
-		<plugin>
```
Comment (fgalan) [inline pom.xml]: Big block of changes... Only indentation changes or any real change?

File: src/main/resources/version.properties [hunk 5]
```diff
@@ -0,0 +1,5 @@
+version=${project.version}
```
Comment (fgalan) [inline src/main/resources/version.properties]: I understand tha these `${...}`'s link this with the content of the pom.xml file. Is that correct?

File: documentation/admin.md (hunk 1 above)
Comment (fgalan) [inline documentation/admin.md]: NTC

File: pom.xml (hunk 3 above)
Comment (crbrox) [inline pom.xml]: A very similar one. A pom.properties (not .xml!) with the same "variables" substituted in the building process.

https://github.com/telefonicaid/fiware-connectors/blob/master/flume/src/main/resources/pom.properties#L5

Comment (fgalan) [inline pom.xml]: NTC

File: pom.xml (hunk 4 above)
Comment (crbrox) [inline pom.xml]: Just indentation ... 
"Source" -> "Format" :-(  (I think it's better now)

Comment (fgalan) [inline pom.xml]: NTC

File: src/main/resources/version.properties (hunk 5 above)
Comment (crbrox) [inline src/main/resources/version.properties]: Correct, those values are substituted by their "values" in pom.xml

Comment (fgalan) [inline src/main/resources/version.properties]: Nice! So the "release stepping process" only need to touch one file :)

NTC

File: documentation/admin.md (hunk 2 above)
Comment (crbrox) [inline documentation/admin.md]: Fixed in 45bad5a 7336ddb

**Comment by fgalan** [comment]
LGTM

**Pull request merged**